whole-tale/girder_wholetale
# Pass only parent as dataSet in AinWT from girder

Fixes an issue where content of directory is too big to be represented as query parameter. Downside is that it introduces another level in `../data/`

### How to test
1. Create a collection with a folder `test`.
2. Import some data (e.g. `/girder/plugins/virtual_resources`) to `test`
3. Run AinWT from `test`
4. Tale should be created with only a single entry in the dataSet (`test`).
